features/step_definitions/shared/tuist.rb.orig

```diff
@@ -0,0 +1,27 @@
+Given(/tuist is available/) do
+  system("swift", "build")
+end
+
+Then(/tuist generates the project/) do
+  system("swift", "run", "tuist", "generate", "--path", @dir)
+  @workspace_path = Dir.glob(File.join(@dir, "*.xcworkspace")).first
+end
+<<<<<<< HEAD
```

Comment:
A merge conflict tissue here.

Reply:
Yeah:) what a score - it's second time this week I push `.orig` files:), am trying to use `git mergetool` (opendiff) this week (instead of IDE to solve conflicts - all in all it's a good tool), but by default it always creates some backup files in the working directory... and then `git add -A` swallow them without noticing, added global ignore - shouldn't happen again. Sorry for that.